src/scheduler.ml

```diff
@@ -727,6 +727,13 @@ let go ?log ?config f =
 
 type exit_or_continue = Exit | Continue
 
+let maybe_clear_screen ~config =
+  let supports_colors = Lazy.force Colors.stderr_supports_colors in
```

Comment:
Instead of doing this, we could extend `Config.adapt_display` to override this option as well

Reply:
Done